### PR TITLE
Added clusterRegistriesNamespace to istio chart.

### PR DESCRIPTION
**Pull request by gyliu513**
/cc @sdake @tiswanso @andraxylia

### CODE REVIEW

File: install/kubernetes/helm/istio/values-istio-auth-multicluster.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ global:
   # Default is 10s second
   refreshInterval: 1s
 
+  # Namespace for ConfigMap which stores clusters configs
+  clusterRegistriesNamespace: istio-system
```
Comment (tiswanso) [inline install/kubernetes/helm/istio/values-istio-auth-multicluster.yaml]: We'd want the default to be the same namespace as the other istio control-plane stuff--`.Release.Namespace`

Comment (gyliu513) [reply install/kubernetes/helm/istio/values-istio-auth-multicluster.yaml]: @tiswanso yes, I saw you have a PR https://github.com/istio/istio/pull/6446 , and once this merged, I think we do not need this PR?

Comment (tiswanso) [reply install/kubernetes/helm/istio/values-istio-auth-multicluster.yaml]: @gyliu513  -- We still need this PR to be able to control this option from helm.  With https://github.com/istio/istio/pull/6446 it just gets taken off the list of multicluster test & doc workflow blockers.

**Pull request closed without merging**